QuizApp/QuizApp/UI/Screens/SolvingQuiz/AnswerView.swift

```diff
@@ -0,0 +1,65 @@
+import UIKit
+
+class AnswerView: UIView {
+
+    weak var delegate: AnswerViewDelegate!
+
+    private var id: Int!
+    private var label: UILabel!
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func set(answer: Answer) {
+        id = answer.id
+        label.text = answer.answer
+    }
+
+}
+
+extension AnswerView: ConstructViewsProtocol {
+
+    func createViews() {
+        label = UILabel()
+        addSubview(label)
+    }
+
+    func styleViews() {
+        layer.cornerRadius = 30
+        backgroundColor = .white.withAlphaComponent(0.3)
+
+        let gesture = UITapGestureRecognizer(target: self, action: #selector(selectedAnswer))
+        addGestureRecognizer(gesture)
+
+        label.font = UIFont(name: "SourceSansPro-Bold", size: 20)
+        label.textColor = .white
+        label.numberOfLines = 0
+    }
+
+    @objc
+    private func selectedAnswer() {
+        delegate.selectedAnswer(with: id ?? 0)
```

Comment:
If the id is not set, that means something has gone wrong in the development. Why is there a default value, it is not declared as an optional?